.github/scripts/rebase-finish.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source `dirname $0`/config.sh
+
+REBASE_BRANCH=$1
+ORIGIN_BRANCH=$2
+
+git fetch --all --prune
+
+# Check if $REBASE_BRANCH exists.
+if ! git show-ref --verify --quiet refs/remotes/origin/$REBASE_BRANCH; then
+  echo "$REBASE_BRANCH does not exist!"
+  exit 1
+fi
+
+git switch $ORIGIN_BRANCH
+git reset --hard origin/$REBASE_BRANCH
```

Comment:
potential loss of sources, maybe we might need a backup for origin for the last rebase and one once a week by creating another branch before finishing rebase.

Reply:
Creating origin branch backup (with timestamp) before force-push is a good idea. I don't understand why another backup once a week is needed though.

Reply:
if something happened at weekend with daily rebase on the second day backup will be overwritten by already broken origin. A weekly backup might help to react and not to lose data or at least minimize impact.

Reply:
OK, my intention was rather to create a unique branch name (with day timestamp) for each rebase backup. Then the weekly backup won't be needed. Deleting old backups manually won't be labor intensive, or alternatively the workflow can delete all branches oder than a week.

Reply:
Yes, it was my initial thought as well. manual clean-up will be fine for now.